.ddev/config.yaml

```diff
@@ -1,39 +1,40 @@
 name: introduction.typo3
 type: typo3
 docroot: .build/public
-php_version: "7.4"
-webserver_type: apache-fpm
+php_version: "8.2"
+webserver_type: nginx-fpm
 router_http_port: "80"
 router_https_port: "443"
 xdebug_enabled: false
 additional_hostnames:
-- docs.introduction.typo3
+    - docs.introduction.typo3
 additional_fqdns: []
 database:
-  type: mariadb
-  version: "10.3"
+    type: mariadb
+    version: "10.4"
```

Comment:
you can use maria 10.6 right away (or 10.11), as this is LTS. https://en.wikipedia.org/wiki/MariaDB#Versioning

Reply:
> you can use maria 10.6 right away (or 10.11), as this is LTS. https://en.wikipedia.org/wiki/MariaDB#Versioning

Currently we have support up to 10.8 but already talked about to include newer LTS versions soon...